raspberry-pi/4/apply-overlays-dtmerge.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, pkgs, stdenvNoCC, dtc, libraspberrypi }:
+
+with lib; {
+  applyOverlays = (base: overlays': stdenvNoCC.mkDerivation {
```

Comment:
maybe make this fine return the function rather than a attrset? see comment on `raspberry-pi/4/pkgs-overlays.nix:8`